ci/upload-docs.sh

```
@@ -54,8 +53,16 @@ git config user.email "nobody@users.noreply.github.com"
 git add --all .
 git commit -q -m"Automatically generated documentation" || exit 0
```

This is silently masking errors in `git commit`; how about:

```bash
git commit ... || {
  echo "'git commit' failed with an error; exiting" >&2
  exit 1    # or 0 if you insist, but it's an error, so why exit with a zero status?
}
```

Thoughts?